docs/developer-guide.rst

```diff
@@ -82,3 +86,39 @@ http://localhost:6543/
 
 Here is it, you're ready to develop in c2cgeoform. Make changes in c2cgeoform,
 run the checks, tests and see the results in c2cgeoform demo application.
+
+Deploy the c2cgeoform_demo on demo server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prepare the demo project:
+
+.. code-block:: shell
+
+   # open a ssh connection with the GMF 2.3 server
+   ssh -A geomapfish-demo.camptocamp.com
+
+   # clone the c2cgeoform repository
+   cd /var/www/vhosts/geomapfish-demo/private
+   git clone git@github.com:camptocamp/c2cgeoform.git
+
+   # generate the c2cgeoform_demo project with mod_wsgi related files
+   APACHE_ENTRY_POINT=c2cgeoform make modwsgi
+
+Create the database as for serving the developement version, see:
```

Comment:
development

Reply:
Create the database as to serve the developement version?